src/components/accessories/dateField/DateField.tsx

```diff
@@ -25,7 +26,13 @@ const DateField: FunctionComponent<IProps> = ({
   const [value, setValue] = useState<Date | null>(null);
 
   useEffect(() => {
-    setValue(fieldValue === "" ? null : new Date(+fieldValue));
+    if (fieldValue === "") {
+      setValue(null);
+    } else if (moment(+fieldValue).isValid()) {
+      setValue(moment(+fieldValue).toDate());
+    } else {
+      setValue(moment(fieldValue).toDate());
+    }
```

Comment:
I continue to not be sure about this. I tested the edit patient locally and it seems like to work as expected with the previous code. Maybe we could restore the previous code and then we could test again which is the problem you reported.

Also, the the if statement it's not properly formatted: you run moment.toDate even if the date is not valid?

Reply:
@axelpezzo Could you please provide the tag of the commit concerning your working code? Because before working on patient form validation, I tested the edit feature and the birthday did'nt get value. So on what commit tag did your code working as exepected? 

For the else statement I agree with you that I should check the date validity.

Reply:
I mean, the previous code I wrote should be correct. The issue here is the date format that comes from the API that is not in millisecond but it's a date string formatted. The issue should be discussed with the back-enders in order to standardize the date format in all his occurency in something of reusable and consistent.

Reply:
The date should always be a timestamp and not a string format date as we already done inside the mockServer API in patientDTO spec.

Reply:
May be you could discuss with Henrique, so as soon as possible I apply what you decide together. Because it is at the moment the only point that is bloking my PR here.

Reply:
I talked with Henrique about this and he agrees with me about this: 

- the API should return only one date format and it should be timestamp averywhere.
- if, the point one it's not possible, this convertion must be outside this componente as for example when you process the date from the API. This component should receive a timestamp in any case.

So, i'm aking to you to restore the previous code and eventually, update the mockData of the opd response with a timestamp as we already done for the patientDTO spec.

Reply:
OK.